visual_tests.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python
```

Comment:
yep, at least in should not belong to that commit... Alos doesn't work right now :-/